Data/Primitive/Array.hs

```diff
@@ -585,7 +583,7 @@ traverseArray f = \ !ary ->
 -- | This is the fastest, most straightforward way to traverse
 -- an array, but it only works correctly with a sufficiently
 -- "affine" 'PrimMonad' instance. In particular, it must only produce
--- *one* result array. 'Control.Monad.Trans.List.ListT'-transformed
+-- __one__ result array. 'Control.Monad.Trans.List.ListT'-transformed
```

Comment:
Maybe `/one/`? Or is that the same?

Reply:
I can also change it to `/one/`, I just thought it makes more sense to actually format the text.